### ISSUE TITLE
[feature request] Allow theme selection per diagram

### ISSUE THREAD

**Issue by jasal82** [terrastruct/d2-obsidian#37]
The theme should be selectable on a per-diagram basis, e.g. by adding a preamble to the d2 code block which contains the settings and is then removed by the plugin before forwarding it to d2.

**Comment by alixander** [comment]
Right. This is built into D2, but I think the Obsidian plugin hasn't been updated to pick those up.

https://d2lang.com/tour/vars/#configuration-variables

**Comment by eriktalvi** [comment]
If all the config-variables could be used . I'm looking at the `layout-engine` variable specifically.
